src/components/Questionnaire/QuestionnaireForm.tsx

```diff
@@ -328,6 +328,25 @@ export function QuestionnaireForm({
     submitBatch({ requests });
   };
 
+  const scrollToQuestion = (questionnaireId: string, groupId?: string) => {
+    setActiveQuestionnaireId(questionnaireId);
+    setActiveGroupId(groupId);
+
+    if (groupId) {
+      const element = document.querySelector(`[data-group-id="${groupId}"]`);
```

Comment:
define element as let outside.

So we can move element check + scroll outside the conditionals.